src/create-repo-relationship.ts

```diff
@@ -1,8 +1,38 @@
-export async function createRepoRelationships(depsList, mainRepo, j1Client, repoName, missingDeps: string[]) {
+// Transform the scope name if the what's in the repo (e.g. Github) does not
+// match what's in npm (other than the `@` sign).
+// For example, TitleCase vs. all lowercase below:
+// - in Github: `JupiterOne/jupiterone-client-nodejs`
+// - in npm: `@jupiterone/jupiterone-client-nodejs`
+const scopeNameTransformations = new Map([
+  ['@jupiterone/', 'JupiterOne/']
+])
+
+export async function createRepoRelationships(depsList: string[], mainRepo, j1Client, repoName, missingDeps: string[]) {
   const successFail = {success: 0, failure: 0, missingDeps: []};
+  const depRegex = /^(@\w+\/)?([^:]+)(:.*)?$/;
   if (depsList.length > 0) {
     for (const dep of depsList) {
-      const fullName = dep.substring(1, dep.indexOf(':'));
+      const match = dep.match(depRegex);
+      if (!match) {
+        continue;
+      }
+
+      const scope = match[1];
+      const repo = match[2];
+
+      let fullName;
+      if (scope && repo) {
+        fullName = scopeNameTransformations.has(scope)
+          ? scopeNameTransformations.get(scope) + repo
+          : scope.substr(1) + repo
```

Comment:
`.substr` is a pseudo deprecated / legacy function, `.substring` and `.slice` are more future proof: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr